Lib/fontTools/feaLib/builder.py

```diff
@@ -1259,18 +1263,23 @@ def build(self):
             self.setLookAheadCoverage_(suffix, st)
             self.setInputCoverage_(glyphs, st)
 
-            st.PosCount = len([l for l in lookups if l is not None])
+            st.PosCount = 0
             st.PosLookupRecord = []
-            for sequenceIndex, l in enumerate(lookups):
-                if l is not None:
-                    if l.lookup_index is None:
-                        raise FeatureLibError('Missing index of the specified '
-                            'lookup, might be a substitution lookup',
-                            self.location)
-                    rec = otTables.PosLookupRecord()
-                    rec.SequenceIndex = sequenceIndex
-                    rec.LookupListIndex = l.lookup_index
-                    st.PosLookupRecord.append(rec)
+            for sequenceIndex, lookupList in enumerate(lookups):
+                if lookupList is not None:
+                    if not isinstance(lookupList, list):
+                        # Can happen with synthesised lookups
```

Comment:
what are "synthesised lookups"?

Reply:
An inline chaining substitution (`sub foo bar' by bar.alt`). But now that I look at it, I wonder if this can be mitigated by returning an array in `get_chained_lookup_`.

Reply:
if possible, I would avoid so many isinstance checks and just enforce that that is a list of lists at the source